README.md

```diff
@@ -31,71 +35,112 @@ Before you start writing _any_ code:
   - Is created by using `rails new .` you will create a new rails app _inside_ of the fork folder instead of creating a _new_ folder for your rails app
-  - Uses [Postgresql](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/08-rails/how-to-use-postgres.md) as it's database.
+    - `rails new` will ask if you want to overwrite the existing files `README.md` and `db/seeds.rb`. Answer `n` to both of these (keep the existing file)
+  - Uses [Postgresql](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/08-rails/how-to-use-postgres.md) as its database.
 
-**Your team shall submit a pull request with a link to your Trello board once you are done with the baseline which shall be reviewed before moving onto implementing the requirements**
+**Your team shall submit a pull request with a link to your Trello board once you are done with this initial setup. This must be reviewed by an instructor before you may begin implementing the requirements.**
+
+### Minimum Requirements
 
-## Minimum Requirements
 However far you and your partner get, the application should have, at a minimum, the following features:
-- Seeded data in your database from the original CSV files
-- Multiple, related, Active Record models
+
+- Seeded data in your database from the original CSV files (see below)
+- Multiple related Active Record models
 - [RESTful routing](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/08-rails/mvc-and-restful-routing.md)
 - Deployed application to [Heroku](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/11-deploying-to-heroku.md)
 - A "look and feel" that will make you and your partner happy!
 
-## Things to Keep in Mind
+### Seeding the Database
+
+We have provided sample data for your site in the form of CSV files. These files are located in the directory `db/seed_data/`. We have also provided a seed script, `db/seeds.rb`, to load these files into your database.
+
+`db/seeds.rb` assumes a few things about how your database is set up:
 
-1.  Try to include useful business logic in your model classes.
-1.  You can include nested routes, but avoid routes that are more than 2-levels deep.
-1.  Use Semantic HTML
-1.  Make good use of partial views
-1.  Don't forget validations! (can a trip exist without a driver or passenger...)
-1.  You may not be able to get all the user stories done.  **Prioritize!** 
+- There are `Driver`, `Passenger` and `Trip` models
+- These models are related in a certain way
+- The field names of these models match the column names in the CSV files
+
+This is the recommended way to configure this project, but not a hard requirement. If you've got a good reason to change the setup, feel free to edit `db/seeds.rb` accordingly.
+
+### Things to Keep in Mind
+
+1. Use git relentlessly
+1. Pair program on difficult problems and to share learning experiences
+1. Deploy early (Wednesday), deploy often.
```

Comment:
Maybe better to just say at least "two days before submission"? So we don't have to update it if the week is different?